pkg/workloadmeta/testing/store.go

```diff
@@ -64,6 +64,44 @@ func (s *Store) ListContainersWithFilter(filter workloadmeta.ContainerFilterFunc
 	return res
 }
 
+// GetProcess implements Store#GetProcess.
+func (s *Store) GetProcess(pid int32) (*workloadmeta.Process, error) {
+	id := string(pid)
+	entity, err := s.getEntityByKind(workloadmeta.KindProcess, id)
+	if err != nil {
+		return nil, err
+	}
+
+	return entity.(*workloadmeta.Process), nil
+}
+
+// ListProcesses implements Store#ListProcesses.
+func (s *Store) ListProcesses() []*workloadmeta.Process {
+	entities := s.listEntitiesByKind(workloadmeta.KindProcess)
+
+	// Not very efficient
+	processes := make([]*workloadmeta.Process, 0, len(entities))
+	for i := range entities {
+		processes = append(processes, entities[i].(*workloadmeta.Process))
+	}
+
+	return processes
+}
+
+// ListProcessesWithFilter implements Store#ListProcessesWithFilter.
+func (s *Store) ListProcessesWithFilter(filter workloadmeta.ProcessFilterFunc) []*workloadmeta.Process {
+	var res []*workloadmeta.Process
+
+	// Not very efficient
```

Comment:
remove `Not very efficient`. this is only way possible in golang

Reply:
this is present in multiple places

Reply:
Indeed, I left it because we also had this comment in the containers code but I still wonder why it was there in the first place.